mac/index.md

```diff
@@ -41,7 +41,12 @@ F# is a strongly-typed functional programming language designed to run on .NET.
 
 ## .NET Core
 
-[.NET Core](https://www.microsoft.com/net/core#macos) is a platform for creating applications that run on Windows, Linux, and Mac. Visual Studio for Mac has support for loading, creating, running, and debugging .NET Core projects.
+[.NET Core](https://www.microsoft.com/net/core#macos) is a platform to create applications that run on Windows, Linux, and Mac. Visual Studio for Mac has support for these .NET Core projects: 
+
```

Comment:
I don't think putting this as a bulleted list makes sense. These are all actions that are supported, not types of projects.

Reply:
Thanks for the feedback. I will remove the bullet list.